### ISSUE TITLE
Training a new model with own data

### ISSUE THREAD

**Issue by priyabiswas12** [Tai-Hsien/MeshSegNet#43]
Hello,

Im interested to train a new model using the pre-trained model. I have my own stl files. May I know if this can be done by labelling the stl file using the "mesh labeler" app you created to create the labeled vtp files, which can then be used to train the model in the steps you illustrated?

Thank you for your time

**Comment by Tai-Hsien** [comment]
Hello,
It is feasible that you prepare your training data by Mesh Lableler and train a model or conduct a transfer learning based on the pre-train model.
However, it's a little bit tricky due to the reasons below:
(1) The number of classes is 15 in this repo, which includes 14 teeth and 1 class for gingiva/background.
(2) The default of number of classes in Mesh Labeler is 17, which includes 16 teeth (including wisdom teeth) and 1 class for gingiva for adult dentition.
Therefore, you have to either customize the colormap.csv in  Mesh Labeler to reduce the number of class to be 15 only, or change the related code in this repo to make the number of classes to be 17.

**Comment by priyabiswas12** [comment]
I see, thank you for the explanation. Im trying to flip the VTP files with label. Can I simply use the mirror function wrt the x axis to flip the mesh or do I need to reverse the labels as well?
Thank you

**Comment by Tai-Hsien** [comment]
Hello,
You also need to adjust the labels.
